packages/verifiable-credentials/src/index.ts

```diff
@@ -1,4 +1,5 @@
 import Credentials from './credentials';
 import SharingCredential from './sharing-credential';
+import * as Utils from './utils'
```

Comment:
These utils are for internal use only. They are only used by the unit tests.

As such, they shouldn't be exposed to users of this library. In reality, those utils should be a part of the unit tests not in the `src/` folder.

Reply:
Noted, I have removed exposed Utils
[commit](https://github.com/verida/verida-js/pull/206/commits/4b4ff64f271e19bacd5211e77cd59f3157be210b)